style.css

```diff
@@ -0,0 +1,88 @@
+#appbar {
+  position: absolute;
+  top: 0;
+  left: 0;
+  background-color: #3c3a39;
+  width: 375px;
+  height: 92px;
+  color: #fff;
+}
+
+#toolbar {
+  position: absolute;
+  bottom: 0;
+  left: 24px;
+  display: flex;
+  height: 40px;
+  width: 327px;
+  align-items: center;
+  justify-content: space-between;
+}
+
+#headline {
+  display: flex;
+  box-sizing: border-box;
+  flex-direction: column;
+  justify-content: center;
+  align-items: flex-start;
+  padding: 120px 24px;
+  position: absolute;
+  width: 375px;
+  height: 590px;
+  left: 0;
+  top: 92px;
+  background-color: #1c1a19;
+}
+
+#header-logo {
+  font-family: 'Roboto', sans-serif;
+  font-size: 18px;
+  line-height: 24px;
+  font-weight: bold;
+}
+
+#main-header {
+  font-family: 'Crete Round', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 56px;
+  line-height: 64px;
+  letter-spacing: 0.37px;
+  color: #ff6b00;
+  margin: 0;
+}
+
+#sub-main-header {
+  height: 27px;
+  font-family: 'Roboto', sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 24px;
+  color: #fff5e1;
+  margin: 12px 0;
+}
+
+#main-paragraph {
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+  color: #fff;
+  margin: 0;
+}
+
+#illustration {
+  position: absolute;
+  bottom: -46px;
+  right: 0;
+}
+
+#social-links {
+  display: flex;
```

Comment:
- [ ] CSS best practices, DRY concept. I think it's a good practice to avoid repeating the same lines of code many times. You can wrap the repetitive code inside a class that you can use whenever you need it in the HTML code. for example `.flex { display: flex;}` 
you can do the same for any repeated code it will upgrade the readability.